src/main/java/space/obminyashka/items_exchange/service/impl/LocationServiceImpl.java

```diff
@@ -123,6 +134,53 @@ public String createParsedLocationsFile(List<RawLocation> creatingData) throws I
         return Files.readString(Path.of(locationInitFilePath).toAbsolutePath(), StandardCharsets.UTF_8);
     }
 
+    @Override
+    public String createParsedAreasFile(List<RawLocation> creatingData) throws IOException {
```

Comment:
Area, District, and City classes have the same fields "nameUA", "nameEN" and that's why you can create some basic class for them both and apply polymorphism instead of having 3 methods with absolutely the same logic